bigtable/admin.go

```diff
@@ -373,6 +384,8 @@ type TableInfo struct {
 	// DEPRECATED - This field is deprecated. Please use FamilyInfos instead.
 	Families    []string
 	FamilyInfos []FamilyInfo
+	// DeletionProtection indicates whether the table is protected against data loss
+	DeletionProtection bool
```

Comment:
Be careful, for NAME_ONLY table view, this field will not be populated in GetTable and ListTables.

We probably should just use DeletionProtection DeletionProtection to be consistent.

We probably should document this a bit here.